src/cript/nodes/exceptions.py

```diff
@@ -57,3 +59,138 @@ def __init__(self, node_type: str, json_dict: str):
 
     def __str__(self):
         return f"JSON Serialization failed for node type {self.node_type} with JSON dict: {self.json_str}"
+
+
+class CRIPTOrphanedNodesError(CRIPTException, ABC):
```

Comment:
possible idea, we could maybe in the future put all orphaned nodes exceptions into one file to keep to keep it cleaner, something like this

`src/cript/nodes/exceptions/exception.py`
`src/cript/nodes/exceptions/orphaned_nodes_exception.py`

Reply:
sure, we can do that.
Not sure we need it now yet.